devtools/test_dashboard/devtools.js

```diff
@@ -160,7 +160,17 @@ setInterval(function() {
 }, 1000);
 
 // Mocks search and plotting
-var f = new Fuse(mocks, {
+var fuse = new Fuse(mocks, {
+    // isCaseSensitive: false,
+    // includeScore: false,
+    // shouldSort: true,
+    // includeMatches: false,
+    // findAllMatches: false,
+    // minMatchCharLength: 1,
+    // location: 0,
+    // threshold: 0.6,
+    // distance: 100,
+    // useExtendedSearch: false,
```

Comment:
Are these comments here for a reason?

Reply:
Simply taken from `fuse.js` docs, illustrating options and their defaults.